docs/operations/rule-configuration.md

```diff
@@ -23,212 +23,242 @@ title: "Retaining or automatically dropping data"
+Interval load rules have type `loadByInterval` and the following example API payload:
 
-Interval load rules are of the form:
 
 ```json
 {
-  "type" : "loadByInterval",
+  "type": "loadByInterval",
   "interval": "2012-01-01/2013-01-01",
   "tieredReplicants": {
     "hot": 1,
-    "_default_tier" : 1
+    "_default_tier": 1
   }
 }
 ```
 
-* `type` - this should always be "loadByInterval"
-* `interval` - A JSON Object representing ISO-8601 Intervals
-* `tieredReplicants` - A JSON Object where the keys are the tier names and values are the number of replicas for that tier.
+Set the following properties:
+- `interval`: a JSON object representing [ISO-8601](https://en.wikipedia.org/wiki/ISO_8601) intervals.
+- `tieredReplicants`: a JSON object containing tier names and the number of replicas for each tier.
 
-### Period Load Rule
+### Period load rule
 
-Period load rules are of the form:
+Period load rules have type `loadByPeriod` and the following example API payload:
 
 ```json
 {
-  "type" : "loadByPeriod",
-  "period" : "P1M",
-  "includeFuture" : true,
+  "type": "loadByPeriod",
+  "period": "P1M",
+  "includeFuture": true,
   "tieredReplicants": {
       "hot": 1,
-      "_default_tier" : 1
+      "_default_tier": 1
   }
 }
 ```
 
-* `type` - this should always be "loadByPeriod"
-* `period` - A JSON Object representing ISO-8601 Periods
-* `includeFuture` - A JSON Boolean indicating whether the load period should include the future. This property is optional, Default is true.
-* `tieredReplicants` - A JSON Object where the keys are the tier names and values are the number of replicas for that tier.
+Set the following properties:
+- `period`: a JSON object representing [ISO-8601](https://en.wikipedia.org/wiki/ISO_8601) periods. The period is from some time in the past to the future or to the current time, depending on the `includeFuture` flag.
+- `includeFuture`: a boolean flag to indicate whether the load period includes the future. Defaults to `true`.
+- `tieredReplicants`: a JSON object containing tier names and the number of replicas for each tier.
 
-The interval of a segment will be compared against the specified period. The period is from some time in the past to the future or to the current time, which depends on `includeFuture` is true or false. The rule matches if the period *overlaps* the interval.
+Druid compares a segment's interval to the period you specify in the rule. The rule matches if the period overlaps the segment interval. 
 
-## Drop Rules
+## Drop rules
 
-Drop rules indicate when segments should be dropped from the cluster.
+Drop rules define when Druid drops segments from the cluster.
 
-### Forever Drop Rule
+### Forever drop rule
 
-Forever drop rules are of the form:
+Forever drop rules have type `dropForever`:
 
 ```json
 {
-  "type" : "dropForever"
+  "type": "dropForever"
 }
 ```
 
-* `type` - this should always be "dropForever"
+Druid drops all segments with this rule from the cluster. 
```

Comment:
Again, move to the top of the section.

Can we explain this? I'm struggling to figure out what "drop forever" means and what would happen if I made this a global rule. Is it maybe used to unload (and prevent reloading) of all segments for one datasource when that datasource is no longer needed? Perhaps as a first step to permanently deleting that datasource? If so, it should only ever be used as a per-datasource rule?

We wait above that load rules are the default. If I have both a load rule and a drop rule for the same datasource, what happens? Will Druid complain? Will it cycle between loading and dropping? Does the load rule overrule the drop rule? Visa versa?